annotation_suite_wrapper.sh

```diff
@@ -127,8 +127,9 @@ function annotateMAF {
     # (1): input file to annotate
     input_file="$1"
     output_file=${ANNOTATED_SUB_DIR_NAME}/$(basename "${input_file}").annotated
+    error_report_file=${OUTPUT_DATA_DIRECTORY}/error_report.txt
     echo -e "\t[INFO] annotateMAF(), annotating MAF: ${input_file} --> ${output_file}"
-    java ${JAVA_SSL_ARGS} -jar ${GENOME_NEXUS_ANNOTATOR_JAR} --filename "${input_file}" --output-filename "${output_file}" --isoform-override ${GENOME_NEXUS_ANNOTATOR_ISOFORM} -p ${GENOME_NEXUS_ANNOTATOR_POST_SIZE} -r
+    java ${JAVA_SSL_ARGS} -jar ${GENOME_NEXUS_ANNOTATOR_JAR} --filename "${input_file}" --output-filename "${output_file}" --isoform-override ${GENOME_NEXUS_ANNOTATOR_ISOFORM} -p ${GENOME_NEXUS_ANNOTATOR_POST_SIZE} -r --error-report-location "${error_report_file}"
```

Comment:
This is a feature I was not familiar with. Looks good.

Also, I see here that we *do* have a reference to JAVA_SSL_ARGS ... and that it points to the scripts directory like this SSL_CERT_PATH=${ANNOTATION_SUITE_SCRIPTS_HOME}/AwsSsl.truststore

So maybe it is the case that we did need to provide to Genie a custom AwsSsl.truststore file previously.  I am not sure why that would be the case now ... but maybe our genie deployment of genome-nexus is using an msk generated SSL certificate which does not validate properly with the default ca_certs file. I don't remember the details, but perhaps this nullifies my comment above regarding AwsSsl.truststore not being needed. I do still think it would be better to put the instructions for generating your own trust store into a separate docs page.

Reply:
Yes, the AwsSsl.truststore is needed specifically for GENIE.  I forget who provided the file to me, but it was provided.